SPDX-FileCopyrightText: 2023 fosslinux <fosslinux@aussies.space>
SPDX-FileCopyrightText: 2022 Simon Marchi <simon.marchi@efficios.com>

SPDX-License-Identifier: GPL-3.0-or-later

Backport of commit 1f237573caa5cf72218ea31fe78eb3983a717ca6

====

Add

  AC_CONFIG_MACRO_DIRS([../config])

So that just running:

  $ autoreconf -vf

... does the right thing (no need to specify -I ../config).

diff --git libiberty/configure.ac libiberty/configure.ac
index 84a7b378fad..28d996f9cf7 100644
--- libiberty/configure.ac
+++ libiberty/configure.ac
@@ -2,6 +2,7 @@ dnl Process this file with autoconf to produce a configure script
 
 AC_INIT
 AC_CONFIG_SRCDIR([xmalloc.c])
+AC_CONFIG_MACRO_DIRS([../config])
 
 # This works around the fact that libtool configuration may change LD
 # for this particular configuration, but some shells, instead of
-- 
2.31.1

